tests/conftest.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from app import create_app
+from app import db
+from flask.signals import request_finished
+from app.models.planet import Planet
+
+
+@pytest.fixture
+def app():
+    app = create_app({'TESTING': True})
+
+    @request_finished.connect_via(app)
+    def expire_session(sender, response, **extra):
+        db.session.remove()
+  
+    with app.app_context():
+        db.create_all()
+        yield app
+
+    with app.app_context():
+        db.drop_all()
+
+
+@pytest.fixture
+def client(app):
+    return app.test_client()
+
+
+@pytest.fixture
+def two_saved_planets(app):
```

Comment:
👍 Consider returning a list/tuple of the two created planets so that a test could know their ids if needed. Whatever the fixture returns is what the corresponding test parameter will have for a value.